### DISCUSSION TITLE
compare inputs with hotkeys

### DISCUSSION THREAD

**Discussion by Kjempe** [ggarra13/mrv2#307] [Q&A] [answered]
Hi, 

Is it possible to compare 2 or more different sources with assigning them hotkeys like 1,2,3..etc instead of clicking them? If not, I think it would be a very useful update:) Thanks!

**Comment by ggarra13** [comment]
I would be somewhat against it, as it would use too many hotkeys.

Currently, you can use Page Up/Page Down to go from one image to the previous/next and loop once it reaches the first/last image.  Is that good enough?

In v1.2.8, you will also be able to use Ctrl+Page Up/Page Down to avoid looping once you reach the first or the last image.

**Comment by ggarra13** [comment]
I've also added now in v1.2.8, the menu entry Image/Go to/"Clip Name".   That way, you can quickly switch from one clip to the next by using the Right Mouse Button menus.

You can test the beta from sourceforge (https://sourceforge.net/projects/mrv2/files/beta/) in about one hour.

**Comment by Kjempe** [comment]
Thanks for the quick answer! Sure, the default hotkeys can work too! Anyway, is there any chance to play every input simultaneously so that I can compare each of them at the same frame? Sorry, if its a dumb question, I just couldn't find a settings for it. 
Thanks in advance! :)

**Reply by ggarra13** [reply]
> Thanks for the quick answer! Sure, the default hotkeys can work too! Anyway, is there any chance to play every input simultaneously so that I can compare each of them at the same frame? Sorry, if its a dumb question, I just couldn't find a settings for it.

Sure!  Go to Panel->Compare (F8 by default).  It will open a panel where you can select a B/C/D/E image and the the type of comparison (Wipe, Tile, Overlay, Difference, etc).  Wipe wipes between two images.  Tile can compare as many images as you want in a Tile.
The way you usually use Wipe is, in the Files Panel (F4), you select the A image.   In the Compare Panel you select the Wipe Button and then the B image.  You can then use Alt + LMB to do a horizontal or vertical wipe.
Note that in v1.2.8 in the sourceforge beta, you can also do all of the above with the Image menu too!

**Comment by Kjempe** [comment]
I got it! My only problem is that I can't compare them with a hotkey in a way that the frame matches on both clips. With next/prev click the playhead's position won't match. Am I missing something?
Thank you again!

**Comment by ggarra13** [comment, marked as answer]
The playhead is stored for each clip.  So if you use Page Up/Page Down, you should first position yourself on each clip on the frame you want to compare.
If you are using "Compare" you can compare "Relative" to the start of each clip or "Absolute" to the actual frame of the "File" clip (A).

**Comment by Kjempe** [comment]
Thanks for your answers! :)
